### ISSUE TITLE
Please update the README!

### ISSUE THREAD

**Issue by DavidAPowers** [localfoodnodes/localfoodnodes#133]
Local Food Nodes is a great idea, but all the information is here: https://localfoodnodes.org/ 
My understanding is you are seeking stars for your repo, and you have your code as an open source project, I think that is great, but it would help a lot if the README actually explained what the project is!!!

**Comment by davidajnered** [comment]
You're right, I make sure to update the readme with some info about the project. Thanks for you input!
